model/project_aliases.go

```diff
@@ -405,6 +406,11 @@ func UpsertAliasesForProject(aliases []ProjectAlias, projectId string) error {
 		}
 		catcher.Add(aliases[i].Upsert())
 	}
+	grip.Debug(message.WrapError(catcher.Resolve(), message.Fields{
```

Comment:
Can we maybe save the results of catcher.Resolve instead of resolving it twice?

Reply:
I don't think it's particularly important since the idea will be to remove this, and this reads cleanly